decidim-core/lib/decidim/core.rb

```diff
@@ -129,6 +129,7 @@ module Decidim
   autoload :HasWorkflows, "decidim/has_workflows"
   autoload :StatsFollowersCount, "decidim/stats_followers_count"
   autoload :StatsParticipantsCount, "decidim/stats_participants_count"
+  autoload :ActionAuthorizationHelper, "decidim/action_authorization_helper"
```

Comment:
I think we can remove this other autoload then 

https://github.com/decidim/decidim/blob/a66edeb4abed33fccb7863651b7180eb25c4eb5c/decidim-core/lib/decidim/view_model.rb#L4

Reply:
And probably we would have the same problem with the ResourceHelper (next line in this same file)